slemicro/README.md

```diff
@@ -444,6 +451,18 @@ operatingSystem:
   users:
     - username: root
       encryptedPassword: $6$ZDh4zjzEsh8K8Svn$DOmn5N2EZZJ1RCys/937tFwID6LfCcCnblp5o0ralWk72a3pmOyTmhsLaHWobBX9mhwVbEBgvKzdudo1jRee3.
+
+$ cat eib/smolvm/network/smolvm1.yaml
+interfaces:
+- name: eth0
+  type: ethernet
+  state: up
+  mac-address: 00:00:00:00:00:00
+  ipv4:
+    dhcp: true
```

Comment:
In my case with dhcp is not working in mac M1 sequoia (just for your info)

Reply:
Can you try with a simple VM created via UTM UI? This works on my machine :D with Sonoma.